ui/src/app/views/invitation-key/component.tsx

```diff
@@ -0,0 +1,109 @@
+import {fullUrl} from '../../utils/fetch';
+
+import {AlertDanger} from 'app/components/alert';
+import {Button} from 'app/components/buttons';
+import {BoldHeader} from 'app/components/headers';
+import {FormInput} from 'app/components/inputs';
+
+import {FetchArgs,
+  InvitationVerificationRequest,
+  ProfileApiFetchParamCreator
+} from 'generated/fetch/api';
+
+import * as React from 'react';
+
+function isBlank(s: string) {
```

Comment:
Instead of this function we could do `!someString.trim()`

Reply:
The problem with that is that it doesn't handle when the string is null or undefined. We were actually originally doing that and it caused issues.